Makefile

```diff
@@ -7,6 +7,9 @@ TAR ?= tar
 ZIP ?= zip
 PLANTUML ?= plantuml # may also be "java -jar plantuml.jar" if installed elsewhere
 
+KCONFIG_CONF ?= conf
+KCONFIG_MCONF ?= mconf
```

Comment:
These commands are rarely available on macOS, so I don't think we should depend on this

Reply:
The alternative would be compile them from source in the project, which I would like to avoid.
There is a good rationale in https://github.com/WangNan0/kbuild-standalone (or use python?)

It is also only needed, if making major changes to the Kconfig file itself. Otherwise, edit `.config`.
The format should be quite straight-forward, even if one needs to comment out - not set as 'n'

Reply:
I guess we should just adopt `vi .config` or `./configure --enable-XXX`

Reply:
Adding support for autoconf would require more infrastructure and generate more boilerplate, imho